community/bolt/src/test/java/org/neo4j/bolt/v1/messaging/example/Nodes.java

```diff
@@ -34,15 +34,33 @@
 {
     public static final Node ALICE = new ValueNode(
             1001L,
-            asList( label( "Person" ), label( "Employee" ) ),
+            asList( Label.label( "Person" ), Label.label( "Employee" ) ),
             map( "name", "Alice", "age", 33L ) );
+
+    static
+    {
+        ALICE = new ValueNode(
+                1001L,
+                asList( Label.label( "Person" ), Label.label( "Employee" ) ),
+                map( "name", "Alice", "age", 33L ) );
+    }
+
     public static final Node BOB = new ValueNode(
             1002L,
-            asList( label( "Person" ), label( "Employee" ) ),
+            asList( Label.label( "Person" ), Label.label( "Employee" ) ),
             map( "name", "Bob", "age", 44L ) );
+
+    static
+    {
+        BOB = new ValueNode(
+                1002L,
+                asList( Label.label( "Person" ), Label.label( "Employee" ) ),
+                map( "name", "Bob", "age", 44L ) );
+    }
+
```

Comment:
You cannot make these assignments in `static { … }` clauses while also having final field assignments at the field definitions.